src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj

```diff
@@ -30,9 +30,23 @@
     <None Include="..\..\README.md" Pack="true" PackagePath="\" Visible="false" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
     <PackageReference Include="Serilog" Version="3.1.0-*" />
-    <!-- The version of this reference must match the major and minor components of the package version prefix. -->
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0-*" />
     <PackageReference Include="Nullable" Version="1.3.1" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(targetframework)' == 'net8.0'">
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0-*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(targetframework)' == 'net7.0'">
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0-*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(targetframework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0-*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(targetframework)' == 'netstandard2.1' or '$(targetframework)' == 'netstandard2.0' or '$(targetframework)' == 'net462'">
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0-*" />
```

Comment:
I'm not exactly sure what should be the version defined here, but at least this does not change the current behavior by using the latest version